PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -848,7 +849,7 @@ void AliAnalysisTaskGammaConvV1::UserCreateOutputObjects(){
     }
     
     if(fIsHeavyIon == 1) 
-      fHistoV0MultVsNumberTPCoutTracks[iCut]    = new TH2F("V0Mult vs TPCout Tracks","V0Mult vs TPCout Tracks",3000,0,6000,3000,0,30000);
+      fHistoV0MultVsNumberTPCoutTracks[iCut]    = new TH2F("V0Mult vs TPCout Tracks","V0Mult vs TPCout Tracks",3000,0,15000,3000,0,40000);
```

Comment:
This histogram will make it nearly impossible to run on PbPb events with more than 1 or 2 wagons in parallel. Please reduce the number of bins as much as possible or only include it once per V0Reader.